app/assets/stylesheets/components/_checklist.scss

```diff
@@ -0,0 +1,195 @@
+
+.floating-button{
```

Comment:
For which checklist is that ? I don't see any checklist anywhere ?

Reply:
Sorry this belongs to the checklist branch. Will remove everything that belongs to the checklist from this branch